app/src/androidTest/java/com/github/se/icebreakrr/ui/sections/NotificationTest.kt

```diff
@@ -107,8 +155,48 @@ class NotificationTest {
   }
 
   @Test
-  fun profilesListNotEmpty() {
-    // TODO: add test for profile cards in the Notification Screen when having meeting request
+  fun InboxListOneElement() {
+    val updatedProfile = profile1.copy(meetingRequestInbox = mapOf(("Jean" to "hello !")))
```

Comment:
why not put `mapOf(("Jean D'eau" to "hello !")))` instead? so that it corresponds to the profile.

Reply:
Yes ! You are right !